bundles/org.openhab.core.audio/src/main/java/org/openhab/core/audio/internal/AudioManagerImpl.java

```diff
@@ -140,20 +140,23 @@ public void play(@Nullable AudioStream audioStream, @Nullable String sinkId, @Nu
                             e.getMessage(), e);
                 }
             }
-            try {
-                sink.process(audioStream);
-            } catch (UnsupportedAudioFormatException | UnsupportedAudioStreamException e) {
-                logger.warn("Error playing '{}': {}", audioStream, e.getMessage(), e);
-            } finally {
-                if (volume != null && oldVolume != null) {
+
+            final PercentType oldVolumeFinal = oldVolume;
+            Runnable toRunWhenProcessFinished = () -> {
+                if (volume != null && oldVolumeFinal != null) {
                     // restore volume only if it was set before
                     try {
-                        sink.setVolume(oldVolume);
+                        sink.setVolume(oldVolumeFinal);
```

Comment:
Some audio sinks will throw `UnsupportedOperationException`, for example the freebox audio sink.
Maybe this exception should be caught and ignored ?

Reply:
Seems legit. Done.